index.html

```diff
@@ -0,0 +1,286 @@
+                    See Project
+                </button>
+                </section>
+        
+            </div>   
+            <!-- card 2 -->
+        
+            <!------------------------------------------------------------->
+        
+            <!-- card 3 -->
+        
+            <div class="card flex">
+        
+                <figure class="fig-image three">
+                </figure>
+        
+                <section class="content">
+                <h2 class="head-two">Tonic</h2>
+        
+                <div class="list flex">
+                    <h3 class="head-three">CANOPY</h3>
+                    <div><img src="images/Counter.svg" alt="point"></div>
+                    <div>Back End Dev</div>
+                    <div><img src="images/Counter.svg" alt="point"></div>
+                    <div>2015</div>
+                </div>
+        
+                <p class="msg">
+                    A daily selection of privately 
+                    personalized reads; no accounts 
+                    or sign-ups required.
+                </p>
+        
+                <ul class="list-two flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+        
+                <button>
+                    See Project
+                </button>
+                </section>
+        
+        
+            </div>
+        
+            <!-- card 3 -->
+        
+            <!------------------------------------------------------------->
+        
+            <!-- card 4 -->
+        
+            <div class="card flex">
+        
+                <figure class="fig-image four">
+                </figure>
+        
+                <section class="content">
+                <h2 class="head-two">Multi-Post Stories</h2>
+        
+                <div class="list flex">
+                    <h3 class="head-three">CANOPY</h3>
+                    <div><img src="images/Counter.svg" alt="point"></div>
+                    <div>Back End Dev</div>
+                    <div><img src="images/Counter.svg" alt="point"></div>
+                    <div>2015</div>
+                </div>
+        
+                <p class="msg">
+                    A daily selection of privately 
+                    personalized reads; no accounts 
+                    or sign-ups required.
+                </p>
+        
+                <ul class="list-two flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+        
+                <button>
+                    See Project
+                </button>
+                </section>
+        
+        
+            </div>
+        
+            <!-- card 4 -->
+        
+            <!---------------------------------------------------------------------------->
+ 
+            <!-- About Me -->
+            <div class="wrapper">
```

Comment:
- [ ] Please note that a `<div` tag does not define a section in HTML. Kindly Change your `<div>` tag to `<section>` as I stated above.